unison-cli/src/Unison/Codebase/Editor/HandleInput.hs

```diff
@@ -2576,10 +2576,10 @@ searchBranchScored names0 score queries =
             pair qn
           HQ.HashQualified qn h
             | h `SH.isPrefixOf` Reference.toShortHash ref ->
-                pair qn
+              pair qn
           HQ.HashOnly h
             | h `SH.isPrefixOf` Reference.toShortHash ref ->
-                Just (Nothing, result)
+              Just (Nothing, result)
```

Comment:
My formatter just did this, I have no idea if its what we want?

Reply:
I'm not sure; we'll try to provide guidance for this in the future.

Reply:
This is the old formatter unfortunately, honestly not a huge deal, but we're preferring to use ormolu v0.4.0.0 if possible.